src/views/viewer/EmbeddedView/EmbeddedView.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router'
+
+import { useAddVideoView, useVideo } from '@/api/hooks'
+import { ViewErrorFallback } from '@/components/ViewErrorFallback'
+import { absoluteRoutes } from '@/config/routes'
+import { useRouterQuery } from '@/hooks/useRouterQuery'
+import { AssetType, useAsset } from '@/providers/assets'
+import { Button } from '@/shared/components/Button'
+import { EmptyFallback } from '@/shared/components/EmptyFallback'
+import { VideoPlayer } from '@/shared/components/VideoPlayer'
+import { SentryLogger } from '@/utils/logs'
+
+import { NotFoundVideoContainer, PlayerSkeletonLoader } from '../VideoView/VideoView.style'
+
+export const EmbeddedView: React.FC = () => {
+  const { id } = useParams()
+  const { loading, video, error } = useVideo(id ?? '', {
+    onError: (error) => SentryLogger.error('Failed to load video data', 'VideoView', error),
+  })
+  const { addVideoView } = useAddVideoView()
+
+  const timestampFromQuery = Number(useRouterQuery('time'))
+
+  const { url: mediaUrl, isLoadingAsset: isMediaLoading } = useAsset({ entity: video, assetType: AssetType.MEDIA })
+
+  const [startTimestamp, setStartTimestamp] = useState<number>()
+  useEffect(() => {
+    const duration = video?.duration ?? 0
+    if (!timestampFromQuery || timestampFromQuery > duration) {
+      return
+    }
+    setStartTimestamp(timestampFromQuery)
+  }, [video?.duration, timestampFromQuery])
```

Comment:
I think we can skip this part in the embedded view

Reply:
I think they mentioned in the call today they wanted the timestamp so this does it already if I got that part right

Reply:
Yeah let's leave it